druid/src/widget/tabs.rs

```diff
@@ -579,7 +580,14 @@ impl<TP: TabsPolicy> TabsBody<TP> {
 impl<TP: TabsPolicy> Widget<TabsState<TP>> for TabsBody<TP> {
     #[instrument(name = "TabsBody", level = "trace", skip(self, ctx, event, data, env))]
     fn event(&mut self, ctx: &mut EventCtx, event: &Event, data: &mut TabsState<TP>, env: &Env) {
-        if event.should_propagate_to_hidden() {
+        if let Event::Notification(notification) = event {
+            if notification.is(SCROLL_TO_VIEW)
+                && Some(notification.route()) != self.active_child(data).map(|w| w.id())
+            {
+                // Ignore SCROLL_TO_VIEW requests from every widget except the active.
```

Comment:
Maybe this could be a property of notifications? Like, say `Notification::ignore_if_hidden`? I'm just trying to think of a way to avoid special-casing `SCROLL_TO_VIEW` in every widget that hides children.

Reply:
I am not sure about this. Then instead of checking whether a `Notification` is `SCROLL_TO_VIEW`, every widget which hides children has to check whether `Notification::ignore_if_hidden` is set. 
Also stopping `Notification`s for hidden widgets is probably pretty unique to `SCROLL_TO_VIEW`.

Reply:
I guess it's unique for now, but I could imagine it not staying that way. But ok, we can punt on this until it actually becomes an issue.